package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "application-manager-web-app",
-  "version": "0.2.0-19",
+  "version": "0.2.0-23",
```

Comment:
Hmmm 
This might be hard when we are merging everything, but not a big deal...

Reply:
That's fine, cos the version number that ends up in develop will be the version number of the last build that was done (from any branch) one all PRs are merged.